cmd/daprd/main.go

```diff
@@ -43,7 +45,7 @@ func main() {
 	// set GOMAXPROCS
 	_, _ = maxprocs.Set()
 
-	rt, err := runtime.FromFlags()
+	rt, err := runtime.FromFlags(os.Args[1:])
```

Comment:
Why are we passing in arguments in this way?

Reply:
This makes it easier to write tests for `runtime.FromFlags`. As @ItalyPaleAle suggested, I'll move it to a separate pr.